### ISSUE TITLE
Electric field saved as integer

### ISSUE THREAD

**Issue by HenningSE** [XENONnT/fuse#131]
I guess we want to have the field as float and not an integer. 

https://github.com/XENONnT/fuse/blob/0c7736a8c28325860a5d28044fe34892fc8bacbd/fuse/plugins/micro_physics/electric_field.py#L28

**Comment by HenningSE** [comment]
Will be fixed with https://github.com/XENONnT/fuse/pull/133

**Comment by mhliu0001** [comment]
In PR #133, the electric field is saved as float16. However, Numba does not support float16. We should instead use float32/float64. Reference: https://stackoverflow.com/questions/59447332/numba-wont-accept-numpy-array-datatype

**Comment by mhliu0001** [comment]
An MWE for this problem would be: 
```python
clusters = st.get_array(run_number, ["microphysics_summary"])
events = st.get_array(run_number, "event_basics")
clusters_in_events = strax.touching_windows(clusters, events, window=10000)
```
Numba will complain about float16.

**Comment by ramirezdiego** [comment]
Fixed in #135.
